package.json

```diff
@@ -1,21 +1,22 @@
 {
   "dependencies": {
```

Comment:
`yarn add` sorted the dependencies. The only change is the addition of react-query